nixos/modules/misc/nixpkgs.nix

```diff
@@ -97,6 +99,8 @@ in
   };
 
   config = {
-    _module.args.pkgs = import ../../.. config.nixpkgs;
+    _module.args.pkgs = _pkgs;
+    # DEPRECATED
+    _module.args.pkgs-i686 = _pkgs.pkgs-i686;
```

Comment:
I dropped the change that deprecated this argument.  It wasn't as simple as putting a trace here, because the attribute set seems to be evaluated and merged even if it's not used in any modules.